tutorial/blending-tutorial/preparing-the-views/README.md

```diff
@@ -0,0 +1,316 @@
+# Declarative Shadow DOM
+
+When blending, we combine view-models and views from different apps. Combining view-models is easy:
+
+```json
+"PetList": {
+    // view-model from PetList 
+},
+"MedicalRecordProvider": {
+    // view-model from MedicalRecordProvider
+}
+```
+
+Combining views is harder because views become useful when arranged in a meaningul way. 
+
+To enable arranging views in meaningful ways, as we will do in the last step [Client-Side Blending](../client-side-blending), we have to define **which elements can be rearranged** and create a **default arrangement** that applies when there are no other views to combine with.
```

Comment:
For me, this could be the first sentence on this page, with everything above moved to Guides (page "Client-side blending")